### PR TITLE
Extra character slots for donors (total of 6)

### PR DESCRIPTION
**Pull request by X-TheDark**
Issue #44 
As per title.

### CODE REVIEW

**Comment by Xantam-13** [comment]
If this works, I'll love you long tine.

**Comment by X-TheDark** [comment]
Oh god, Travis is already on 510, huh?

**Comment by X-TheDark** [comment]
Travis issue caused by it using 510 version to compile, which doesn't, due to Regex changes.

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1198,4 +1201,6 @@ var/global/list/special_roles = list( //keep synced with the defines BE_* in set
 			hardset_dna(character, null, null, null, null, chosen_species, features)
 		else
 			character.update_body()
-			character.update_hair()
\ No newline at end of file
+			character.update_hair()
+			
+#define DONOR_CHARACTER_SLOTS
```
Comment (TehFlaminTaco) [inline code/modules/client/preferences.dm]: This should be undef, Not #define

**Pull request merged**